src/utils/customErrors.js

```diff
@@ -1,15 +1,17 @@
-let serverErrorList = {
+const serverErrorList = {
   EACCES: "You don't have access to bind the server to port {port}.",
   EADDRINUSE: 'There is already a process listening on port {port}.'
 };
 
 function serverErrors(err, port) {
```

Comment:
Code was previously built to handle unknown errors `if (!desc) ...`, but an exception on the `.replace` call would prevent this code path from ever executing. 

Exposed by tests, fixed.